karma.conf.js

```diff
@@ -1,45 +1,74 @@
+const fs = require('fs');
+const minimist = require('minimist');
+
 module.exports = function(config) {
-  config.set({
-    basePath: '',
-    frameworks: ['mocha'],
-    browsers: ['Chrome', 'ChromeCanary', 'Safari', 'Firefox'],
-    reporters: ['progress'],
-    client: {
-      mocha: {
-        reporter: 'html',
+  // Default config for all packages
+  let opts = {
+      basePath: '',
+      frameworks: ['mocha'],
+      browsers: ['Chrome', 'ChromeCanary', 'Safari', 'Firefox'],
+      reporters: ['progress'],
+      client: {
+        mocha: {
+          reporter: 'html',
+        },
       },
-    },
-    files: [
-      'packages/*/src/**/__tests__/**',
-    ],
-    exclude: [
-      '**/*.map',
-    ],
-    preprocessors: {
-      '**/*.ts': ['webpack'],
-      '**/*.js': ['webpack'],
-    },
-    webpack: {
-      devtool: 'eval',
-      resolve: {
-        extensions: ['.js', '.ts'],
+      exclude: [
+        '**/*.map',
+      ],
+      preprocessors: {
+        '**/*.ts': ['webpack'],
+        '**/*.js': ['webpack'],
       },
-      module: {
-        loaders: [
-          {
-            test: /\.tsx?$/, loader: 'ts-loader?transpileOnly=true',
-          },
-        ],
+      webpack: {
+        devtool: 'eval',
+        stats: 'errors-only',
+        resolve: {
+          extensions: ['.js', '.ts'],
+        },
+        module: {
+          loaders: [
+            {
+              test: /\.tsx?$/, loader: 'ts-loader?transpileOnly=true',
+            },
+          ],
+        },
       },
-    },
-    mime: {
-      'text/x-typescript': ['ts', 'tsx'],
-    },
-    port: 9876,
-    colors: true,
-    logLevel: config.LOG_INFO,
-    autoWatch: true,
-    singleRun: false,
-    concurrency: Infinity,
-  });
+      webpackMiddleware: {
+        stats: {
+          chunks: false,
+        },
+      },
+      mime: {
+        'text/x-typescript': ['ts', 'tsx'],
+      },
+      port: 9876,
+      colors: true,
+      logLevel: config.LOG_INFO,
+      autoWatch: true,
+      singleRun: false,
+      concurrency: Infinity,
+  };
+
+  const argv = minimist(process.argv);
+  if (argv.only) {
+    // Run tests for a specific package
+    if (!fs.existsSync(`./packages/${argv.only}`)) {
```

Comment:
Style nit: template literals are easier to read if you pad the inside with spaces: `${ argv.only }`